pkg/fees/txFees.test.ts

```diff
@@ -0,0 +1,51 @@
+import { getFeesBTC } from "./txFees"
```

Comment:
this is in pkg/fees and the file name does not tell the user about the blockchain this is for.

Reply:
Adjusted